src/Contracts/Http/ResponsesInterface.php

```diff
@@ -16,7 +16,7 @@
  * limitations under the License.
  */
 
-use \Neomerx\JsonApi\Document\Error;
+use \Neomerx\JsonApi\Contracts\Document\ErrorInterface;
```

Comment:
Yeah, this is different from how PHP Storm orders the use statements. I'm also not used to having to put `\` at the start of a `use` statement.